examples/azure/backup/build.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env nash
+
+import klb/azure/login
+import klb/azure/group
+import klb/azure/nic
+import klb/azure/subnet
+import klb/azure/vm
+import klb/azure/availset
+import klb/azure/storage
+import klb/azure/disk
+import klb/azure/vnet
+import klb/azure/nsg
+import klb/azure/route
+import klb/azure/snapshot
+import config.sh
+
+fn addsuffix(name) {
+	# Providing true uniqueness with the limits on the names is pretty hard :-)
+	s <= head -n1 /dev/urandom | md5sum | tr -dc A-Za-z0-9 | cut -b 1-10
+
+	return $name+"-"+$s
+}
+
+fn create_subnet(name, cidr) {
+	azure_nsg_create($name, $group, $location)
+	azure_subnet_create($name, $group, $vnet, $cidr, $name)
+	azure_route_table_create($name, $group, $location)
+
+	hoptype = "Internet"
+
+	route <= azure_route_table_route_new("default", $group, $name, "0.0.0.0/0", $hoptype)
+
+	azure_route_table_route_create($route)
+}
+
+fn new_vm_nodisk(name, subnet) {
+	# create ssh key
+	accessdir = "/tmp/.config/ssh/"
+	accesskey = $accessdir+"id_rsa-"+$name
+
+	-test -e $accesskey
```

Comment:
Sorry, did not have update nash super powers and this example is heavily copied from a previous one :-)